### ISSUE TITLE
[bug] Android tv integration randomly disconnects 

### ISSUE THREAD

**Issue by jkoking** [unfoldedcircle/bug-reports#283]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

At seemingly random times after I push a button that should send a command to my android tv device (nvidia shield pro) it will not receive the command and then refuses to send any more commands until I reboot the device. The IR functionality still works when the remote is in this state. 

### How to Reproduce

1. launch an activity containing an android tv device
2. wait an uncertain amount of time and press a button or repeatedly press a button (I haven't been able to determine an exact procedure to reproduce the issue)

### Expected behavior

The remote should not randomly disconnect from android tv devices

### System version

1.6.1

### Additional context

[UCR2_logs_2024-01-18.txt](https://github.com/unfoldedcircle/feature-and-bug-tracker/files/13982571/UCR2_logs_2024-01-18.txt)

**Comment by martonborzak** [comment]
Please provide logs from the Android TV integration.

**Comment by jkoking** [comment]
When I narrow the logs to just the Android TV integration, the file is blank. So, maybe it's not an issue with the Android TV integration?

**Comment by albaintor** [comment]
Hi, same here. It happens frequently, I have to reboot the remote to make the integration work again.
It looks like the same problem as the appletv last week when the state was not updated and commands were ignored:
[UCR2_logs_2024-03-24.zip](https://github.com/unfoldedcircle/feature-and-bug-tracker/files/14735042/UCR2_logs_2024-03-24.zip)

See the most recent lines at the beginning : all the cmd_id are ignored although they are received by intg-androidtv driver
I am on the last firmware release 1.7.6
```
2024-03-24 09:47:32.617768 +00:00	intg-androidtv	INFO	DEBUG:ucapi.api:[('127.0.0.1', 50804)] ->: {"kind": "resp", "req_id": 42, "code": 200, "msg": "result", "msg_data": {}}
2024-03-24 09:47:32.615163 +00:00	intg-androidtv	INFO	INFO:driver:Got 00044BE87F74 command request: home None
```
This command (home) for ex was not processed.

**Comment by zehnm** [comment]
Reconnection logic has been refactored and improved (https://github.com/unfoldedcircle/integration-androidtv/issues/28).
New Android TV integration version will soon be released in beta channel.

**Comment by zehnm** [comment]
Reconnection fixes are included in the just released beta firmware 1.7.9 and will be part of the next public release.

**Comment by jkoking** [comment]
@zehnm  For me the bug still occurs on the beta but i do get a device "unavailable error" which is new.

Update: as I was writing this comment, I checked the remote again and it seems to have fixed itself.

[UCR2_logs_2024-04-05.txt](https://github.com/unfoldedcircle/feature-and-bug-tracker/files/14878527/UCR2_logs_2024-04-05.txt)

**Comment by zehnm** [comment]
Fixed in release 1.7.12
